tgui/src/interfaces/ftl_navigation.ract

```diff
@@ -1,12 +1,16 @@
 {{#if data.screen == 0}}
+	<ui-button icon='info-circle' action='shipinf' state='{{data.can_jump ? null : "selected"}}' >Ship Information</ui-button>
```

Comment:
I'm not going to review ract files, this is a mess in the first place and if you've made changes that work then it's fine